### ISSUE TITLE
GameGuru MAX - Request - All MAX Content as DLCs

### ISSUE THREAD

**Issue by MonkeyFrogStudio** [TheGameCreators/GameGuruRepo#4479]
GameGuru MAX comes with a ton for free content, which is fantastic. However, there are a few of us that want to use only our own custom assets in MAX. As a result, the over 12GB of models in entitybank in the Steam install area is just needlessly taking up hard drive space. Sure, I could delete all that content, but the next update of MAX would just put it right back. Therefore, for those of us who want to only use our own content, we're stuck with all this stuff we're never going to use eating up our hard drive space. 

The solution, to me, would be to make all that content a DLC. Personally, I would like to see each part it's own DLC. That way, the Arctic Collection, Basement Collection, etc., would all be their own DLCs. Completely free, of course, just like the Particle Editor is. This way, the end-user could download only what they want at the time and, later, if they choose, disable it, just like we can with any DLC.

I would recommend the same for the demos, too (i.e. make them DLCs). They don't really take up much space, but I don't need them. Since the demos require objects from the stock MAX assets, then when someone goes to add the DLC it could warn them what else is needed to be downloaded (or perhaps it would allow for a checkbox that would auto-download the other stuff needed to complete the demo).

If you're worried about MAX using too much hard drive space (as was indicated in the discussion about Project Folders), then there needs to be a way for the end-user to NOT have 12+GB of stuff sitting on their hard drive (unused for people like me).

Speaking of Project Folders, if a project would copy over everything used in a project (not just referenced it), then the end-user could use something from the MAX stock assets and, if they are DLCS, remove them from their hard drive and ONLY have the one or two models they are using left on their hard drives, in their project folder instead of having to have 12+GB of stuff left in their hard drive just to use those few objects.

I realize that not everyone wants to or can create their one objects for use in MAX. For those people, they could just tick to download the DLCs they want and be done with it. For the rest of us? Well, we could free up that hard drive space for our own content. ;)

**Comment by davetgc** [comment]
@LeeBamberTGC - Possibly make them Gift Packs?

**Comment by MonkeyFrogStudio** [comment]
Or just give them their own category and call them "Stock Asset DLCs"?

**Comment by Kravenwolf** [comment]
This would be a great option.

I don't use any of the stock assets and I only have around 40GB of space left on my SSD. Because of them, I had to install MAX on my slower HDD so all of those extra gigs of media that I don't even use can quite literally just sit there and do nothing.

Though, I'd recommend bundling them all together as a single DLC. Separating them into multiple DLCs would break all of the demos since they all use assets from multiple stock folders.

**Comment by MonkeyFrogStudio** [comment]
> Though, I'd recommend bundling them all together as a single DLC. Separating them into multiple DLCs would break all of the demos since they all use assets from multiple stock folders.

Yes, true. However, if actual Project Folders worked, then downloading a demo would download it's own project folder and that demo would ONLY have in it the objects used. In that case, you'd not even need to download the stock objects to check out a demo. :)

**Comment by Blood-Moon-Interactive** [comment]
Great suggestion @MonkeyFrogStudio
